src/docker-entrypoint.sh

```diff
@@ -63,7 +64,12 @@ git_commit() {
     echo "::debug file=entrypoint.sh,line=54 No files changed, skipping commit"
     exit 0
   else
-    git commit -m "${GIT_COMMIT_MESSAGE}"
+    local signoff
+    signoff=""
+    if [ "${GIT_PUSH_SIGN_OFF}" = "true" ]; then
+      signoff="-s"
```

Comment:
This -s shoould not be -S in uppercase. 
lowercase s measn silent not signed.

Reply:
No, it should be [`-s, --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--s). There's no `-S` in git, AFAIR, and for that matter there's no _silent_ mode, rather [`--quiet`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---quiet) mode.